src/main/java/org/springframework/samples/petclinic/system/PetClinicToggles.java

```diff
@@ -13,15 +13,26 @@
 
     public static Toggle toggleFindOwnerByLastName = new Toggle(Toggle.ALWAYS_ON_WHEN_ENABLED);
     public static Toggle toggleListOfOwners = new Toggle(0.5f);
+    public static Toggle toggleFindOwnerByTelephone = new Toggle(0.5f);
+
+
+    // TODO remove
+    public static Collection<Boolean> toggles = new ArrayList<>(Arrays.asList(toggleFindOwnerByLastName.isOn(), toggleFindOwnerByTelephone.isOn(), toggleListOfOwners.isOn()));
     public static Toggle toggleFindOwnerByLocation = new Toggle(0.65f);
     public static Toggle toggleFindOwnerByFirstName = new Toggle(0.5f);
 
 
+
+
     public static Collection<Boolean> getToggleValues() {
         return new ArrayList<>(Arrays.asList(
```

Comment:
fix order should be lastName, listOfOwners, firstName, Location, thenTelephone

Remove duplicates too